linux_os/guide/system/logging/package_rsyslog-module-gtls_installed/rule.yml

```diff
@@ -0,0 +1,39 @@
+documentation_complete: true
```

Comment:
This new rule is basically the same of `package_rsyslog-gnutls_installed`. The same purpose and descriptions. The only difference is the package name. So, I propose to use the existing rule and simply update the package name for the `sle` products when calling the template. eg: `pkgname@sle15: rsyslog-module-gtls`.

This would avoid duplicated rules and content.

Reply:
I have reverted the changes locally and testing it.

But during past testing the test was ignoring the changed package name for SLES. This is the reason why I created a new rule.

Maybe an oversight on my end.

Reply:
Worked as expected. Can't remember what my error there was. Also added macros to update the descriptions.